packages/zend-form/library/Zend/Form/Element/File.php

```diff
@@ -104,7 +104,7 @@ public function loadDefaultDecorators()
      */
     public function setPluginLoader(Zend_Loader_PluginLoader_Interface $loader, $type)
     {
-        $type = strtoupper($type);
+        $type = is_string($type) ? strtoupper($type) : $type;
```

Comment:
idem

Reply:
I will leave this one as it calls `parent::setPluginLoader($type)` and potentially that could be called with null and handled there.